enterprise/frontend/src/metabase-enterprise/collections/utils.ts

```diff
@@ -3,7 +3,7 @@ import { REGULAR_COLLECTION } from "./constants";
 
 export function isRegularCollection({
   authority_level,
-}: Bookmark | Collection) {
+}: Bookmark | Partial<Collection>) {
```

Comment:
search results don't have the whole collection, and you don't actually need the whole collection to know whether it's a regular collection